src/index.ts

```diff
@@ -0,0 +1,5 @@
+import CliAnalytics from './analytics/cli-analytics';
+
+export {
```

Comment:
I think you can change this to `export CliAnalytics`. This will enable you to change this `const CliAnalytics = require('cli-analytics-sdk').CliAnalytics;` to `const CliAnalytics = require('cli-analytics-sdk');`